code/components/citizen-scripting-mono-v2/src/MonoScriptRuntime.cpp

```diff
@@ -263,11 +263,12 @@ int MonoScriptRuntime::HandlesFile(char* filename, IScriptHostWithResourceData*
 	}
 
 	// last supported date for this pilot of mono_rt2, in UTC
-	constexpr int maxYear = 2023, maxMonth = 8, maxDay = 31;
+	constexpr int maxYear = 2023, maxMonth = 11, maxDay = 31;
```

Comment:
November has 30 days 😏 Let's put this to December (12) instead, too much going on right now.